cloudbreak/config/config.go

```diff
@@ -121,6 +121,74 @@ func configRead(c *cli.Context) error {
 	return nil
 }
 
+// CheckConfigAndCommandFlagsDP : CAAS does not require workspace check
+func CheckConfigAndCommandFlagsDP(c *cli.Context) error {
+	err := fl.CheckRequiredFlagsAndArguments(c)
+	if err == nil {
+		return configReadDP(c)
+	}
+	return err
+}
+
+func configReadDP(c *cli.Context) error {
```

Comment:
this function is almost identical to the configRead method, seems like unnecessary code replication

Reply:
Since workspace is not mandatory for caas. this function is (configRead - workspace).